R/polars-package.R

```diff
@@ -2,7 +2,9 @@
 "_PACKAGE"
 
 ## usethis namespace: start
-#' @importFrom utils globalVariables head tail download.file capture.output str
+#' @importFrom stats na.omit median
```

Comment:
I do not know if this is necessary.

Reply:
We add methods for those so I guess they should be imported

Reply:
Other functions like mean are not imported but seem to work.

Reply:
Yes but `mean()` is in `base` and not in `stats` (and I guess the other functions as well)

Reply:
Oh, I see. Thanks.